cli/tools/task.rs

```diff
@@ -59,9 +60,10 @@ pub async fn execute_script(
     let seq_list = deno_task_shell::parser::parse(&script)
       .with_context(|| format!("Error parsing script '{task_name}'."))?;
     let env_vars = collect_env_vars();
-    let exit_code =
-      deno_task_shell::execute(seq_list, env_vars, &cwd, Default::default())
-        .await;
+    let local = LocalSet::new();
+    let future =
+      deno_task_shell::execute(seq_list, env_vars, &cwd, Default::default());
+    let exit_code = local.run_until(future).await;
```

Comment:
Why is `LocalSet` required here? Is it coming from `deno_task_shell` typings? If so I will update `deno_task_shell` to not require it if possible (CC @dsherret)

Reply:
https://github.com/denoland/deno_task_shell/blob/f70223aa98593751dd83b3397fbdc469145b4850/src/shell/mod.rs#LL118C41-L118C52

Reply:
Actually, thinking about it - it's probably not that bad that we use `LocalSet` for this specific purpose.

Reply:
I think we should pass an Executor in there.